src/components/dashboard/navigation.js

```diff
@@ -27,17 +27,19 @@ class Navigation extends Component {
   render() {
     return (
       <div className="mainContainer" data-test={ifDev("navigation")}>
+        <div className='navContainer'>
         <div className="Bars">
           <FontAwesomeIcon className="bars-icon" icon={faBars} />
           <i className="far fa-bars"></i>
         </div>
 
         <div className="Links">
-          <NavLink className="Groa NavLink" to="/">
+          <NavLink className="Groa-NavLink" to="/">
```

Comment:
Why was this changed to Groa-NavLink?

Reply:
Ben changed this for styling purposes to set the Home link apart, following an earlier spec of the design file, can be changed to whatever going forward